client/js/components/pages/TeamsPage.js

```diff
@@ -29,18 +29,17 @@ class TeamsPage extends Component {
       );
     }
 
-    let teamsList;
-    if (teamProfiles.length) {
-      teamsList = (
-        <section>
-          {teamProfiles}
-        </section>
-      );
+    if (!teamProfiles.length) {
+      teamProfiles.push(<p className="alert error">Something gone wrong, none profile found.</p>);
```

Comment:
BTW, I don't think we need that. When application will be set up from the scratch, there won't be any teams. There is also possibility to delete teams, so I'd say "no teams" situation is still a correct one and does need an error message.